mars/deploy/oscar/tests/test_cmdline.py

```diff
@@ -179,7 +182,7 @@ def test_cmdline_run(supervisor_args, worker_args, use_web_addr):
         for proc in w_procs + [sv_proc]:
             if not proc:
                 continue
-            proc.terminate()
+            proc.send_signal(signal.SIGINT)
```

Comment:
May not perform as expected in Windows.

Reply:
@keyile Why change this line?

Reply:
@wjsi @keyile I will revert this line to pass the Windows CI.